database/migrations/2023_02_24_155104_create_products_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->float('price');
```

Comment:
I know this is just a mock exercise but avoid using float/double for prices, especially in the DB. We (corsi.it) also learnt it the hard way

https://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency

Reply:
Thanks , I will count that .